recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,290 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
```

Comment:
If we can make this generic for all Python versions we can use this, if not we must remove it.

Reply:
I used conda skeleton to create the initial version of this file, so I'm not really sure what this is doing.

Reply:
That is fine. We diverge a little bit from conda-skeleton but that is still a good starting point.